assets/javascripts/parent_group_overview.js

```diff
@@ -0,0 +1,92 @@
+/* jshint multistr: true */
+
+function setupParentGroupOverviewAssets(group_id) {
+    var cookie_product_name = '_product_' + group_id + '_grouped_by';
+    var cookie_all_products_name = 'product_all_grouped_by';
+
+    function setCookie(name, value) {
+        document.cookie = name + "=" + value;
+    }
+
+    function getCookie(name) {
+        var value = "; " + document.cookie;
+        var parts = value.split("; " + name + "=");
+        if (parts.length == 2) return parts.pop().split(";").shift();
+    }
+
+    function update_grouped_by_default_links() {
+        if (window.location.hash !== getCookie(cookie_product_name))
+            $('#grouped_by_default').show();
+        else
+            $('#grouped_by_default').hide();
+
+        if (window.location.hash !== getCookie(cookie_all_products_name))
+            $('#grouped_by_default_all').show();
+        else
+            $('#grouped_by_default_all').hide();
+    }
+
+    function update_grouped_by_classes() {
+        function removeClasses(id, classes){
```

Comment:
So `document.getElementById(id).classList.remove(classes)` doesn't work? `remove` should accept multiple parameters. The same counts for `add`.

Reply:
It could be removed/added with only one call but not as array of classes, only a different parameters
https://www.w3schools.com/jsref/prop_element_classlist.asp
Refactored that

Reply:
> not as array of classes, only a different parameters

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax